clients/ui/frontend/src/app/pages/modelCatalog/utils/modelCatalogUtils.ts

```diff
@@ -235,9 +216,8 @@ const isFilterIdInMap = (
   filters: CatalogFilterOptions,
 ): filterId is keyof CatalogFilterOptions => typeof filterId === 'string' && filterId in filters;
 
-// TODO tech debt: different filterQuery syntax is needed depending on whether the API stores an array of values or a single string value.
-//   the current filter_options API response does not indicate the difference between these two types of fields, so for now we hard-code them.
-const KNOWN_ARRAY_FILTER_IDS: (keyof CatalogFilterOptions)[] = [
+// Array-based fields that need LIKE syntax instead of IN syntax for database queries
+const ARRAY_FIELD_FILTER_IDS: (keyof CatalogFilterOptions)[] = [
```

Comment:
This should no longer be necessary actually, but just in case we can save that for its separate tech debt ticket. I can't find the backend fix right now 😅